server/lib/nl/detection/types.py

```diff
@@ -428,4 +428,18 @@ class Detection:
   svs_detected: SVDetection
   classifications: List[NLClassifier]
   llm_resp: Dict = field(default_factory=dict)
-  detector: ActualDetectorType = ActualDetectorType.HybridHeuristic
\ No newline at end of file
+  detector: ActualDetectorType = ActualDetectorType.HybridHeuristic
+
+
+@dataclass
+class DetectionArgs:
+  # Name of the embeddings index type to override ENV default
+  embeddings_index_type: str
+  # Query mode. e.g., `strict`, `toolformer`
+  mode: str
+  # Reranker model name
+  reranker: str
+  # Enable entities/triples detection (e.g., for biomed DC)
+  allow_triples: bool
+  # Disable stop-word removal for query detection
+  disable_stop_words: bool
```

Comment:
nit: include_stop_words? (currently should be: disable stop words removal)

Reply:
Done.